test-website/template/package.json

```
@@ -0,0 +1,14 @@
+{
+  "author": "@directus-labs",
+  "description": "Based on Simple Website CMS (https://github.com/directus-labs/directus-templates)",
+  "directusTemplate": true,
+  "files": [
+    "src"
+  ],
+  "name": "directus-template-simple-website-cms-ve",
+  "templateName": "Simple Website CMS VE",
+  "version": "1.0.0-ve",
+  "scripts": {
+    "setup-directus": "source ../simple-cms/nuxt/.env && npx directus-template-cli@latest apply -p --directusUrl=$DIRECTUS_URL --templateLocation='.' --templateType='local' --directusToken=$DIRECTUS_SERVER_TOKEN"
+  }
```

```suggestion
  "scripts": {
    "setup-directus": "env-cmd -f ../simple-cms/nuxt/.env -x -- npx directus-template-cli@latest apply -p --directusUrl=\"$DIRECTUS_URL\" --templateLocation=\".\" --templateType=\"local\" --directusToken=\"$DIRECTUS_SERVER_TOKEN\""
  },
  "devDependencies": {
    "env-cmd": "10.1.0"
  }
```

`source .env` and `$DIRECTUS_URL` was causing issues in Windows because `source` is not an existing command and variable interpolation uses a different format `%DIRECTUS_URL%`. The `env-cmd` package smooths both of these over nicely on my end.

Hope it still works on your end with these changes 😄